RecoTracker/LSTCore/standalone/code/core/write_sdl_ntuple.h

```diff
@@ -4,55 +4,55 @@
 #include <iostream>
 #include <tuple>
 
-#include "SDLMath.h"
+#include "lstMath.h"
 #include "Event.h"
 
 #include "AnalysisConfig.h"
 #include "trkCore.h"
 #include "AccessHelper.h"
 
-using SDLEvent = SDL::Event<ALPAKA_ACCELERATOR_NAMESPACE::Acc3D>;
+using lstEvent = lst::Event<ALPAKA_ACCELERATOR_NAMESPACE::Acc3D>;
```

Comment:
type names should be capitalized
```suggestion
using LstEvent = lst::Event<ALPAKA_ACCELERATOR_NAMESPACE::Acc3D>;
```
or `LSTE...`